client/components/learn/Resources/resources-model.ts

```diff
@@ -137,6 +147,11 @@ export const RESOURCES: Record<ResourceCategory, ResourceDescription[]> = {
       url: 'https://openmoji.org/',
       cover: `${process.env.CLOUDFRONT_PUBLIC}/public/images/resources/openmoji_500w.png`,
     },
+    {
+      title: 'Ionic Icons',
+      url: 'https://ionic.io/ionicons',
+      cover: 'https://ionic.io/ionicons/assets/img/meta/ionicons-og.png',
```

Comment:
Same feedback as above: copy this photo in our GIT and link to our CDN - we want to self-host.